src/memcached-promisify.js

```diff
@@ -0,0 +1,120 @@
+// forked from https://github.com/flightstats/memcached-promisify
+
+import { assign } from 'lodash';
+import Memcached from 'memcached';
+
+const MAX_EXPIRATION = 60 * 60 * 24 * 30; // memcached's max exp in seconds (30 days)
+
+function cachePromise(client, method, ...args) {
+    return new Promise((resolve, reject) => {
+        client._cache[method](...args, (err, data) => { // eslint-disable-line no-underscore-dangle
+            if (err) {
+                reject(err);
+            } else {
+                resolve(data);
+            }
+        });
+    });
+}
+
+/**
+ * Cache
+ * @constructor
+ * @param {Object} config - cache key prefix
+ * @param {string} [config.keyPrefix] - cache key prefix
+ * @param {string} [config.cacheHost] - cache host url
+ * @param {Object} [options] - options passed to memcached
+ *
+ * @example
+ *    new Cache('prefix', 'host');
+ *
+ */
+class Cache {
+    constructor(config, options = { maxExpiration: MAX_EXPIRATION }) {
```

Comment:
Something is strange about this configuration:
* We have both `config` and `options`
* The default `MAX_EXPIRATION` is const but `localhost:11211` is not